version.yml

```diff
@@ -1,6 +1,6 @@
 ---
 variables:
   MAJOR: 6
-  MINOR: 2
-  PATCH: 1
+  MINOR: 3
+  PATCH: 0
```

Comment:
```suggestion
  PATCH: 1
```

Should this be "1" (based on https://mattermost.atlassian.net/browse/DOPS-776)?

Reply:
Ok, sounds like it needs a separate PR.

Reply:
yeah that was my intention, separate PR for 6.3.1

Reply:
yes, we should release 6.3.0 and then 6.3.1